xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/menus_content.vm

```diff
@@ -217,7 +217,7 @@
   ##
   ## Display the menu only if there is some content
   ##
-  #displayMenu('tmAdminActions', 'cog', $menuContent)
+  #displayMenu('tmAdminActions', 'cog', $menuContent, 'admin.page')
```

Comment:
If the page will be terminal there will be confusions since the tooltip will say "Administer Page" while the first action would be "Administer Parent". 

You should use "core.menu.actions"